framework/13-inch/common/intel.nix

```diff
@@ -5,6 +5,7 @@
 
   boot.kernelParams = [
     # Fixes a regression in s2idle, making it more power efficient than deep sleep
+    # Does not really make a difference for me in 12th-gen-intel - Linux 6.8 (SebTM - 04/2024) with/without 0.X Watt usage
     "acpi_osi=\"!Windows 2020\""
```

Comment:
possibly the s2idle regression has been fixed

Reply:
Did you also test deep vs s2idle power consumption with/without this change as well?

Reply:
@RaitoBezarius Updated based on your comment and will do testing after rebuild/reboot again and add it here.